tests/ar_test.toit

```diff
@@ -34,6 +34,7 @@ write-ar file-mapping/Map --add-with-ar-file/bool=false:
   writer := io.Buffer
   ar-writer := ArWriter writer
   file-mapping.do: |name content|
+    if content is string: content = content.to-byte-array
```

Comment:
Why is this necessary?

Reply:
good catch.
should have in in the next line.
done.